docs/data/material/getting-started/installation/installation.md

```diff
@@ -52,7 +52,7 @@ pnpm add @mui/material @mui/styled-engine-sc styled-components
 </codeblock>
 
 :::warning
-Visit the [Styled engine guide](/material-ui/guides/styled-engine/) for more information about how to configure styled-components.
+Visit the [styled-components guide](/material-ui/guides/styled-components/) for more information on configuration.
```

Comment:
What about updating the content of this callout to be about _“Using styled-components as an engine at this moment is not working when used in a SSR projects...”_ instead?

Seems better to know that sooner, before going to a "how to configure" page. (Same for the @mui/system installation page)

Reply:
not a bad idea!